The job **`x86_64-gnu-llvm-16`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/7624216633/job/20765906767) [(plain)](https://github.com/rust-lang/rust/commit/863b6f45892f2a20987bc86249dcb663b5322479/checks/20765906767/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
GITHUB_ACTION=__run_7
GITHUB_ACTIONS=true
GITHUB_ACTION_REF=
GITHUB_ACTION_REPOSITORY=
GITHUB_ACTOR=weihanglo
GITHUB_API_URL=https://api.github.com
GITHUB_BASE_REF=master
GITHUB_ENV=/home/runner/work/_temp/_runner_file_commands/set_env_0d7e9cb4-8f7f-4c45-9b17-7c8d2309ef9d
GITHUB_EVENT_NAME=pull_request
GITHUB_EVENT_NAME=pull_request
GITHUB_EVENT_PATH=/home/runner/work/_temp/_github_workflow/event.json
GITHUB_GRAPHQL_URL=https://api.github.com/graphql
GITHUB_HEAD_REF=split-dward-kind-lto
GITHUB_JOB=pr
GITHUB_PATH=/home/runner/work/_temp/_runner_file_commands/add_path_0d7e9cb4-8f7f-4c45-9b17-7c8d2309ef9d
GITHUB_REF=refs/pull/120264/merge
GITHUB_REF_NAME=120264/merge
GITHUB_REF_PROTECTED=false
---
GITHUB_SERVER_URL=https://github.com
GITHUB_SHA=ff31d24968562b5cb02b547493d250d1a5420a81
GITHUB_STATE=/home/runner/work/_temp/_runner_file_commands/save_state_0d7e9cb4-8f7f-4c45-9b17-7c8d2309ef9d
GITHUB_STEP_SUMMARY=/home/runner/work/_temp/_runner_file_commands/step_summary_0d7e9cb4-8f7f-4c45-9b17-7c8d2309ef9d
GITHUB_TRIGGERING_ACTOR=weihanglo
GITHUB_WORKFLOW_REF=rust-lang/rust/.github/workflows/ci.yml@refs/pull/120264/merge
GITHUB_WORKFLOW_SHA=ff31d24968562b5cb02b547493d250d1a5420a81
GITHUB_WORKSPACE=/home/runner/work/rust/rust
GOROOT_1_19_X64=/opt/hostedtoolcache/go/1.19.13/x64
---
Built container sha256:9c3c93a371e5aed5c18185b24f130d95d5140dbd72a9b325e7b6b49e521a4faa
Looks like docker image is the same as before, not uploading
https://ci-caches.rust-lang.org/docker/7ebc15c01a233894034d277c8cce4e949f4e7791f66b4727c8fb6e058a0b8171d6152e1441d677cef0653843ceeee469c097b8699b2bb74249e674f6aa1a8813
sha256:9c3c93a371e5aed5c18185b24f130d95d5140dbd72a9b325e7b6b49e521a4faa
Setting extra environment values for docker:  --env ENABLE_GCC_CODEGEN=1 --env GCC_EXEC_PREFIX=/usr/lib/gcc/
[CI_JOB_NAME=x86_64-gnu-llvm-16]
##[group]Clock drift check
  local time: Tue Jan 23 10:19:20 UTC 2024
  network time: Tue, 23 Jan 2024 10:19:20 GMT
  network time: Tue, 23 Jan 2024 10:19:20 GMT
##[endgroup]
sccache: Starting the server...
##[group]Configure the build
configure: processing command line
configure: 
configure: build.configure-args := ['--build=x86_64-unknown-linux-gnu', '--llvm-root=/usr/lib/llvm-16', '--enable-llvm-link-shared', '--set', 'rust.thin-lto-import-instr-limit=10', '--set', 'change-id=99999999', '--enable-verbose-configure', '--enable-sccache', '--disable-manage-submodules', '--enable-locked-deps', '--enable-cargo-native-static', '--set', 'rust.codegen-units-std=1', '--set', 'dist.compression-profile=balanced', '--dist-compression-formats=xz', '--set', 'build.optimized-compiler-builtins', '--disable-dist-src', '--release-channel=nightly', '--enable-debug-assertions', '--enable-overflow-checks', '--enable-llvm-assertions', '--set', 'rust.verify-llvm-ir', '--set', 'rust.codegen-backends=llvm,cranelift,gcc', '--set', 'llvm.static-libstdcpp', '--enable-new-symbol-mangling']
configure: target.x86_64-unknown-linux-gnu.llvm-config := /usr/lib/llvm-16/bin/llvm-config
configure: llvm.link-shared     := True
configure: rust.thin-lto-import-instr-limit := 10
configure: change-id            := 99999999
---
##[endgroup]
Testing GCC stage1 (x86_64-unknown-linux-gnu -> x86_64-unknown-linux-gnu)
   Compiling y v0.1.0 (/checkout/compiler/rustc_codegen_gcc/build_system)
    Finished release [optimized] target(s) in 1.37s
     Running `/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-codegen/x86_64-unknown-linux-gnu/release/y test --use-system-gcc --use-backend gcc --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/stage1-tools/cg_gcc --release --no-default-features --mini-tests --std-tests`
Using system GCC
Using system GCC
[BUILD] example
[AOT] mini_core_hello_world
/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-tools/cg_gcc/mini_core_hello_world
abc
---
---- [run-make] tests/run-make/split-debuginfo stdout ----

error: make failed
status: exit status: 2
command: cd "/checkout/tests/run-make/split-debuginfo" && env -u CARGO_MAKEFLAGS -u MAKEFLAGS -u MFLAGS -u RUSTFLAGS AR="ar" CC="cc -ffunction-sections -fdata-sections -fPIC -m64" CXX="c++ -ffunction-sections -fdata-sections -fPIC -m64" HOST_RPATH_DIR="/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib" LD_LIB_PATH_ENVVAR="LD_LIBRARY_PATH" LLVM_BIN_DIR="/usr/lib/llvm-16/bin" LLVM_COMPONENTS="aarch64 aarch64asmparser aarch64codegen aarch64desc aarch64disassembler aarch64info aarch64utils aggressiveinstcombine all all-targets amdgpu amdgpuasmparser amdgpucodegen amdgpudesc amdgpudisassembler amdgpuinfo amdgputargetmca amdgpuutils analysis arm armasmparser armcodegen armdesc armdisassembler arminfo armutils asmparser asmprinter avr avrasmparser avrcodegen avrdesc avrdisassembler avrinfo binaryformat bitreader bitstreamreader bitwriter bpf bpfasmparser bpfcodegen bpfdesc bpfdisassembler bpfinfo cfguard codegen core coroutines coverage debuginfocodeview debuginfodwarf debuginfogsym debuginfologicalview debuginfomsf debuginfopdb demangle dlltooldriver dwarflinker dwarflinkerparallel dwp engine executionengine extensions filecheck frontendhlsl frontendopenacc frontendopenmp fuzzercli fuzzmutate globalisel hexagon hexagonasmparser hexagoncodegen hexagondesc hexagondisassembler hexagoninfo instcombine instrumentation interfacestub interpreter ipo irprinter irreader jitlink lanai lanaiasmparser lanaicodegen lanaidesc lanaidisassembler lanaiinfo libdriver lineeditor linker loongarch loongarchasmparser loongarchcodegen loongarchdesc loongarchdisassembler loongarchinfo lto m68k m68kasmparser m68kcodegen m68kdesc m68kdisassembler m68kinfo mc mca mcdisassembler mcjit mcparser mips mipsasmparser mipscodegen mipsdesc mipsdisassembler mipsinfo mirparser msp430 msp430asmparser msp430codegen msp430desc msp430disassembler msp430info native nativecodegen nvptx nvptxcodegen nvptxdesc nvptxinfo objcarcopts objcopy object objectyaml option orcjit orcshared orctargetprocess passes perfjitevents powerpc powerpcasmparser powerpccodegen powerpcdesc powerpcdisassembler powerpcinfo profiledata remarks riscv riscvasmparser riscvcodegen riscvdesc riscvdisassembler riscvinfo riscvtargetmca runtimedyld scalaropts selectiondag sparc sparcasmparser sparccodegen sparcdesc sparcdisassembler sparcinfo support symbolize systemz systemzasmparser systemzcodegen systemzdesc systemzdisassembler systemzinfo tablegen target targetparser textapi transformutils ve veasmparser vecodegen vectorize vedesc vedisassembler veinfo webassembly webassemblyasmparser webassemblycodegen webassemblydesc webassemblydisassembler webassemblyinfo webassemblyutils windowsdriver windowsmanifest x86 x86asmparser x86codegen x86desc x86disassembler x86info x86targetmca xcore xcorecodegen xcoredesc xcoredisassembler xcoreinfo xray" LLVM_FILECHECK="/usr/lib/llvm-16/bin/FileCheck" NODE="/usr/bin/node" PYTHON="/usr/bin/python3" RUSTC="/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc" RUSTDOC="/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustdoc" RUST_BUILD_STAGE="stage2-x86_64-unknown-linux-gnu" S="/checkout" TARGET="x86_64-unknown-linux-gnu" TARGET_RPATH_DIR="/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib/rustlib/x86_64-unknown-linux-gnu/lib" TMPDIR="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo" "make"
Build completed unsuccessfully in 0:35:42
Build completed unsuccessfully in 0:35:42
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features foo.rs -g -C  split-debuginfo=off
[ ! -f /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp ]
[ ! -f /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo ]
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features foo.rs -g
[ ! -f /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp ]
[ ! -f /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo ]
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features foo.rs -g  -C split-debuginfo=packed -Zsplit-dwarf-kind=split
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo && exit 1 || exit 0
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo.dwp
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features foo.rs -g  -C split-debuginfo=packed -Zsplit-dwarf-kind=single
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo && exit 1 || exit 0
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo.dwp
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features baz.rs -g  -Csplit-debuginfo=packed -Zsplit-dwarf-kind=split \
 --crate-type=rlib -Clinker-plugin-lto
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp && exit 1 || exit 0
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/libbaz.rlib
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features baz.rs -g  -Csplit-debuginfo=packed -Zsplit-dwarf-kind=single \
 --crate-type=rlib -Clinker-plugin-lto
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp && exit 1 || exit 0
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/libbaz.rlib
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features  -C split-debuginfo=packed -C debuginfo=2 \
 -Z split-dwarf-kind=split --remap-path-prefix /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo=/a foo.rs -g
objdump -Wi /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo | grep DW_AT_GNU_dwo_name | (! grep /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo) || exit 1
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo && exit 1 || exit 0
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo.dwp
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features  -C split-debuginfo=packed -C debuginfo=2 \
 -Z split-dwarf-kind=single --remap-path-prefix /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo=/a foo.rs -g
objdump -Wi /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo | grep DW_AT_GNU_dwo_name | (! grep /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo) || exit 1
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo && exit 1 || exit 0
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo.dwp
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features  -C split-debuginfo=packed -C debuginfo=2 \
 -Z split-dwarf-kind=single --remap-path-prefix /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo=/a \
 -Z remap-path-scope=split-debuginfo-path foo.rs -g
objdump -Wi /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo | grep DW_AT_GNU_dwo_name | (! grep /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo) || exit 1
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo && exit 1 || exit 0
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo.dwp
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features  -C split-debuginfo=packed -C debuginfo=2 \
 -Z split-dwarf-kind=single --remap-path-prefix /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo=/a \
 -Z remap-path-scope=macro foo.rs -g
objdump -Wi /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo | grep DW_AT_GNU_dwo_name | (grep /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo) || exit 1
    <14>   DW_AT_GNU_dwo_name: (indirect string, offset: 0x29): /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo.foo.cba44757bc0621b9-cgu.0.rcgu.o
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo && exit 1 || exit 0
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo.dwp
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features --crate-type lib  -C split-debuginfo=packed \
 -Zsplit-dwarf-kind=split -C debuginfo=2 -g bar.rs
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.rlib
/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/libbar.rlib
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp && exit 1 || exit 0
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features --extern bar=/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/libbar.rlib  \
 -C split-debuginfo=packed -Zsplit-dwarf-kind=split -C debuginfo=2 -g main.rs
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo && exit 1 || exit 0
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/main.dwp
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/main
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features --crate-type lib  -C split-debuginfo=packed \
 -Zsplit-dwarf-kind=single -C debuginfo=2 -g bar.rs
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.rlib
/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/libbar.rlib
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp && exit 1 || exit 0
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features --extern bar=/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/libbar.rlib  \
 -C split-debuginfo=packed -Zsplit-dwarf-kind=single -C debuginfo=2 -g main.rs
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo && exit 1 || exit 0
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/main.dwp
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/main
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features foo.rs -g  -C split-debuginfo=unpacked -Zsplit-dwarf-kind=split
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp && exit 1 || exit 0
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features foo.rs -g  -C split-debuginfo=unpacked -Zsplit-dwarf-kind=single
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o
/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo.foo.cba44757bc0621b9-cgu.0.rcgu.o
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo && exit 1 || exit 0
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp && exit 1 || exit 0
rm /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo
LD_LIBRARY_PATH="/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo:/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0-bootstrap-tools/x86_64-unknown-linux-gnu/release/deps:/checkout/obj/build/x86_64-unknown-linux-gnu/stage0/lib" '/checkout/obj/build/x86_64-unknown-linux-gnu/stage2/bin/rustc' --out-dir /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo -L /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo  -Ainternal_features baz.rs -g  -Csplit-debuginfo=unpacked -Zsplit-dwarf-kind=split \
 --crate-type=rlib -Clinker-plugin-lto
ls /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o && exit 1 || exit 0
/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo.foo.cba44757bc0621b9-cgu.0.rcgu.o
--- stderr -------------------------------
--- stderr -------------------------------
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp': No such file or directory
objdump: Warning: Unable to load dwo file: /a/foo.foo.cba44757bc0621b9-cgu.0.rcgu.dwo
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo': No such file or directory
objdump: Warning: Unable to load dwo file: /a/foo.foo.cba44757bc0621b9-cgu.0.rcgu.o
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo': No such file or directory
objdump: Warning: Unable to load dwo file: /a/foo.foo.cba44757bc0621b9-cgu.0.rcgu.o
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo': No such file or directory
objdump: Warning: Unable to load dwo file: /checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/foo.foo.cba44757bc0621b9-cgu.0.rcgu.o
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp': No such file or directory
warning: unused import: `Bar`
##[warning] --> main.rs:3:11
  |
3 | use bar::{Bar, make_bar};
  |
  = note: `#[warn(unused_imports)]` on by default

warning: 1 warning emitted
warning: 1 warning emitted

ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp': No such file or directory
warning: unused import: `Bar`
##[warning] --> main.rs:3:11
  |
3 | use bar::{Bar, make_bar};
  |
  = note: `#[warn(unused_imports)]` on by default

warning: 1 warning emitted
warning: 1 warning emitted

ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.o': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwo': No such file or directory
ls: cannot access '/checkout/obj/build/x86_64-unknown-linux-gnu/test/run-make/split-debuginfo/split-debuginfo/*.dwp': No such file or directory
make: *** [Makefile:245: unpacked-lto-split] Error 1



failures:
```

</details>

--- a/tests/run-make/split-debuginfo/Makefile
+++ b/tests/run-make/split-debuginfo/Makefile
@@ -234,28 +234,28 @@ unpacked-single:
 	ls $(TMPDIR)/*.dwp && exit 1 || exit 0
 	rm $(TMPDIR)/$(call BIN,foo)
 
-unpacked-lto: packed-lto-split packed-lto-single
+unpacked-lto: unpacked-lto-split unpacked-lto-single
 
 # - rmeta file added to rlib, no object files are generated and thus no debuginfo is generated
-# - `.o` never created
+# - `.o` present (bitcode)
 # - `.dwo` never created
 # - `.dwp` never created
 unpacked-lto-split:
 	$(RUSTC) baz.rs -g $(UNSTABLEOPTS) -Csplit-debuginfo=unpacked -Zsplit-dwarf-kind=split \
 		--crate-type=rlib -Clinker-plugin-lto
-	ls $(TMPDIR)/*.o && exit 1 || exit 0
+	rm $(TMPDIR)/*.o
 	ls $(TMPDIR)/*.dwo && exit 1 || exit 0
 	ls $(TMPDIR)/*.dwp && exit 1 || exit 0
 	rm $(TMPDIR)/libbaz.rlib
 
 # - rmeta file added to rlib, no object files are generated and thus no debuginfo is generated
-# - `.o` never created
+# - `.o` present (bitcode)
 # - `.dwo` never created
 # - `.dwp` never created
 unpacked-lto-single:
 	$(RUSTC) baz.rs -g $(UNSTABLEOPTS) -Csplit-debuginfo=unpacked -Zsplit-dwarf-kind=single \
 		--crate-type=rlib -Clinker-plugin-lto
-	ls $(TMPDIR)/*.o && exit 1 || exit 0
+	rm $(TMPDIR)/*.o
 	ls $(TMPDIR)/*.dwo && exit 1 || exit 0
 	ls $(TMPDIR)/*.dwp && exit 1 || exit 0
 	rm $(TMPDIR)/libbaz.rlib

